### PR TITLE
Precious/optimus page

### PR DESCRIPTION
**Pull request by atepem**
## Proposed changes

Added content to Optimus page:
* Agent metrics (coming soon) section 
* What is Olas Optimus
* Why Olas Optimus

## Screenshots/Recordings

image
image
image

## Things to keep in mind

- [ ] I confirm I have updated the meta title and description for the page, if applicable.

### CODE REVIEW

File: components/OptimusPage/WhatIsOlasOptimus.jsx [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    <p className="mb-3">Every 24 hours, Optimus follows a simple cycle:</p>
+    <ol className="list-decimal">
+      <li className="mb-2">
+        <strong>Finds the best opportunities:</strong>
+        <p>
+          It looks for liquidity pools on{' '}
+          <a
+            href="https://merkl.angle.money/"
+            className="text-purple-600 underline"
+          >
```
Comment (truemiller) [inline components/OptimusPage/WhatIsOlasOptimus.jsx:95]:
```suggestion
          <a
            href="https://merkl.angle.money/"
            className="text-purple-600 underline"
            target="_blank"
          >
```

File: components/OptimusPage/WhatIsOlasOptimus.jsx [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+          </li>
+        </ol>
+      </li>
+      <li>
+        <strong>Tracks performance:</strong>
+        <p>
+          By keeping track of the number of transactions performed on the
+          Optimism chain, Optimus contributes to your Key Performance Indicators
+          (KPIs) for Olas staking rewards. This means that if you run Optimus
+          daily, you earn staking rewards based on your agent’s activity.
```
Comment (truemiller) [inline components/OptimusPage/WhatIsOlasOptimus.jsx:126]:
```suggestion
          , targeting those on Balancer and Uniswap with an APR (annual percentage
          rate) higher than 5%.
        </p>
      </li>
      <li>
        <strong>Makes smart investments:</strong>
        <ol className="list-[lower-alpha] mb-2 ml-4">
          <li className="py-1">
            If it&apos;s the first investment and the pool&apos;s Annual
            Percentage Rate (APR) is over 5%, Optimus adds liquidity.
          </li>
          <li className="py-1">
            If it finds a pool with a higher APR than your current one, Optimus
            moves your assets to maximize your earnings.
          </li>
          <li className="py-1">
            {' '}
            If no opportunity is available, it waits until the next period and
            restarts the cycle.
          </li>
        </ol>
      </li>
      <li>
        <strong>Tracks performance:</strong>
        <p>
          By keeping track of the number of transactions performed on the
          Optimism chain, Optimus contributes to your Key Performance Indicators
          (KPIs) for Olas staking rewards. This means that if you run Optimus
          daily, you earn staking rewards based on your agent’s activity.
```

File: components/OptimusPage/WhatIsOlasOptimus.jsx [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+          , targeting those on Balancer and Uniswap with an Annual Percentage
+          Rate (APR) higher than 5%.
+        </p>
+      </li>
+      <li>
+        <strong>Makes smart investments:</strong>
+        <ol className="list-[lower-alpha] mb-2 ml-4">
+          <li className="py-1">
+            If it&apos;s the first investment and the pool&apos;s Annual
+            Percentage Rate (APR) is over 5%, Optimus adds liquidity.
```
Comment (truemiller) [inline components/OptimusPage/WhatIsOlasOptimus.jsx:107]: the "explainer" should be in the brackets here
```suggestion
            If it&apos;s the first investment and the pool&apos;s APR (annual
            percentage rate) is over 5%, Optimus adds liquidity.
```

File: components/OptimusPage/WhatIsOlasOptimus.jsx [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+        <p>
+          It looks for liquidity pools on{' '}
+          <a
+            href="https://merkl.angle.money/"
+            className="text-purple-600 underline"
+          >
+            Merkl↗
+          </a>
+          , targeting those on Balancer and Uniswap with an Annual Percentage
+          Rate (APR) higher than 5%.
```
Comment (truemiller) [inline components/OptimusPage/WhatIsOlasOptimus.jsx:99]: better to the explain the abbreviation, and reuse APR in later lines
```suggestion
          , targeting those on Balancer and Uniswap with an APR (annual percentage
          rate) higher than 5%.
```

File: components/OptimusPage/WhatIsOlasOptimus.jsx [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+            Percentage Rate (APR) is over 5%, Optimus adds liquidity.
+          </li>
+          <li className="py-1">
+            If it finds a pool with a higher APR than your current one, Optimus
+            moves your assets to maximize your earnings.
+          </li>
+          <li className="py-1">
+            {' '}
+            If no opportunity is available, it waits until the next period and
+            restarts the cycle.
```
Comment (truemiller) [inline components/OptimusPage/WhatIsOlasOptimus.jsx:116]: reads better as below; Optimus checks multiple opportunities
```suggestion
            If no opportunities are available, it waits until the next period and
            restarts the cycle.
```

File: components/OptimusPage/WhatIsOlasOptimus.jsx [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+          </li>
+        </ol>
+      </li>
+      <li>
+        <strong>Tracks performance:</strong>
+        <p>
+          By keeping track of the number of transactions performed on the
+          Optimism chain, Optimus contributes to your Key Performance Indicators
+          (KPIs) for Olas staking rewards. This means that if you run Optimus
```
Comment (truemiller) [inline components/OptimusPage/WhatIsOlasOptimus.jsx:125]: similar to APR change (above)
```suggestion
          Optimism chain, Optimus contributes to your KPIs
          (key performance indicators) for Olas staking rewards. This means that if you run Optimus
```

File: components/OptimusPage/WhyOlasOptimus.jsx [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+];
+
+export const OptimusFooter = () => (
+  <SectionWrapper
+    customClasses="py-12 border border-purple-200 border-x-0"
+    backgroundType="SUBTLE_GRADIENT"
+  >
+    <div className="grid max-w-screen-xl xl:gap-0 lg:px-12 mx-auto items-center">
+      <h3 className="text-center w-full italic text-purple-900 font-medium">
+        Join the revolution in AI Agent-powered DeFi Management with Olas
```
Comment (truemiller) [inline components/OptimusPage/WhyOlasOptimus.jsx:36]:
```suggestion
        Join the revolution in AI agent-powered DeFi management with Olas
```

File: components/OptimusPage/WhyOlasOptimus.jsx [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+    <div
+      className="max-w-screen-xl lg:px-44 mx-auto lg:gap-8 xl:gap-0 lg:grid-cols-12"
+      id="why-olas-optimus"
+    >
+      <h2
+        className={`${SUB_HEADER_LG_CLASS} text-left mb-8 lg:text-center lg:mb-14`}
+      >
+        Why Olas Optimus?
+      </h2>
+      <InfoCardList wrapperClasses="" cards={list} />
```
Comment (truemiller) [inline components/OptimusPage/WhyOlasOptimus.jsx:54]:
```suggestion
      <InfoCardList cards={list} />
```

File: components/OptimusPage/WhatIsOlasOptimus.jsx [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+);
+
+const AgentTypes = () => (
+  <>
+    <h2 className="text-left text-2xl font-semibold tracking-tight my-3">
+      Agent types
+    </h2>
+    <div className="grid grid-cols-1 md:grid-cols-2 gap-4 md:gap-0 my-6">
+      {types.map((item, index) => {
+        let borderClassName = 'pr-6';
```
Comment (truemiller) [inline components/OptimusPage/WhatIsOlasOptimus.jsx:48]: no need to redefine this every map iteration

File: components/OptimusPage/WhatIsOlasOptimus.jsx [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+        if (index % 2 !== 0) borderClassName = ' md:border-l-1.5 pl-6';
```
Comment (truemiller) [inline components/OptimusPage/WhatIsOlasOptimus.jsx:49]: padding is having undesirable effects on mobile (see below)
can use `gap-...` with `grid` or `flex` to seperate elements without retaining padding when responsive

image

File: components/OptimusPage/WhatIsOlasOptimus.jsx [hunk 11]
```diff
@@ -0,0 +1,139 @@
...
+  },
+];
+
+const WhatIsOptimus = () => (
+  <div>
+    <h1 className={`${SUB_HEADER_CLASS} font-semibold text-left text-4xl mb-8`}>
+      What is Olas Optimus?
+    </h1>
+    <p className="mb-20">
+      Olas Optimus agent economy is a multi-agent system of autonomous AI agents
```
Comment (truemiller) [inline components/OptimusPage/WhatIsOlasOptimus.jsx:32]:
```suggestion
      The Olas Optimus agent economy is a multi-agent system of autonomous AI agents
```
or
```suggestion
      Olas Optimus' agent economy is a multi-agent system of autonomous AI agents
```

File: components/OptimusPage/WhatIsOlasOptimus.jsx [hunk 12]
```diff
@@ -0,0 +1,139 @@
...
+    id: 'Trader',
+    src: '/images/predict-page/traders.png',
+    desc: 'Finds the best DeFi opportunities and makes smart investments.',
+    linkText: 'More about Optimus Trader agent',
+    link: '/services/babydegen',
+  },
+  {
+    id: 'Mech',
+    src: '/images/predict-page/mechs.png',
+    desc: 'Operating as Olas Mech agent economy, Mechs provide intelligence for Trader agents for managing DeFi assets.',
```
Comment (truemiller) [inline components/OptimusPage/WhatIsOlasOptimus.jsx:20]:
```suggestion
    desc: 'Operating as the Olas Mech agent economy, Mechs provide intelligence for Trader agents for managing DeFi assets.',
```

File: components/OptimusPage/OptimusAgentMetrics.jsx [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+      <Card className={`${CARD_BG} max-h-[250px] p-16`}>
+        <Image
+          alt="Optimus metrics"
+          src="/images/optimus-page/optimus-metrics.svg"
+          height={48}
+          width={48}
+          className="mb-6 mx-auto"
+        />
+        <p className="text-slate-500">
+          Olas Optimus agent economy metrics are coming soon.
```
Comment (truemiller) [inline components/OptimusPage/OptimusAgentMetrics.jsx:21]: less passive, better for alert messages like these
```suggestion
          Olas Optimus agent economy metrics coming soon.
```

File: components/OptimusPage/WhyOlasOptimus.jsx [hunk 14]
```diff
@@ -0,0 +1,64 @@
+import { SECTION_BOX_CLASS, SUB_HEADER_LG_CLASS } from 'common-util/classes';
+import { InfoCardList } from 'components/InfoCardList';
+import SectionWrapper from 'components/Layout/SectionWrapper';
+import { BicepsFlexed, HandCoins, Sparkles, Target } from 'lucide-react';
+
+const list = [
+  {
+    title: 'Innovation',
+    icon: <Sparkles color="#606F85" />,
+    desc: 'Harnesses the power of autonomous agents to optimize asset management in DeFi.',
```
Comment (truemiller) [inline components/OptimusPage/WhyOlasOptimus.jsx:10]:
```suggestion
    desc: 'Harnesses the power of autonomous agents to optimize DeFi asset management.',
```

File: components/OptimusPage/WhyOlasOptimus.jsx [hunk 15]
```diff
@@ -0,0 +1,64 @@
...
+  },
+  {
+    title: 'Novel UX',
+    icon: <BicepsFlexed color="#606F85" />,
+    desc: "Users don't need to understand the underlying DeFi primitives. Their agent takes care of the complexities.",
+  },
+  {
+    title: 'Profit Potential',
+    icon: <HandCoins color="#606F85" />,
+    desc: 'Operators can run trader agents to potentially earn staking rewards and engage in autonomous DeFi asset management.',
```
Comment (truemiller) [inline components/OptimusPage/WhyOlasOptimus.jsx:20]:
```suggestion
    desc: 'Operators run trader agents to earn staking rewards and engage in autonomous DeFi asset management.',
```

File: pages/agent-economies/optimus.jsx [hunk 16]
```diff
@@ -1,12 +1,18 @@
 import PageWrapper from 'components/Layout/PageWrapper';
 import Meta from 'components/Meta';
+import { OptimusAgentMetrics } from 'components/OptimusPage/OptimusAgentMetrics';
 import { OptimusHero } from 'components/OptimusPage/OptimusHero';
+import { WhatIsOlasOptimus } from 'components/OptimusPage/WhatIsOlasOptimus';
+import { WhyOlasOptimus } from 'components/OptimusPage/WhyOlasOptimus';
 
 const Optimus = () => (
   <PageWrapper>
     <Meta pageTitle="Optimus" description="AI Agent-powered DeFi Management" />
```
Comment (truemiller) [inline pages/agent-economies/optimus.jsx:10]:
```suggestion
    <Meta pageTitle="Optimus" description="AI agent-powered DeFi management" />
```

File: components/OptimusPage/WhatIsOlasOptimus.jsx (hunk 10 above)
Comment (atepem) [reply components/OptimusPage/WhatIsOlasOptimus.jsx:49]: would it be bad practice if I just added md breakpoints 😅 ?

**Review by truemiller — APPROVED** [review]

**Review by Tanya-atatakai — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**